examples/src/examples/animation/component-properties.mjs

```diff
@@ -296,6 +296,7 @@ async function example({ canvas, deviceType, data, assetPath, glslangPath, twgsl
 class ComponentPropertiesExample {
     static CATEGORY = 'Animation';
     static WEBGPU_ENABLED = true;
+    static DESCRIPTION = 'This example demonstrates how to use the Anim Component to animate the properties of other Components.';
```

Comment:
For now I have only added a description to this example. Let me know if the code is correct so I can add text to more examples that aren't clear from their name.